build/googleAnalyticsDonationFormExtension.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array('react', 'wp-element'), 'version' => 'c66c2da91ed4468a8ac0');
```

Comment:
@JoshuaHungDinh we need to add the build folder to `.gitignore` and delete this folder so this folder does not get included in the repo.

Reply:
Yes! I thought that it looked off. Resolved: b48f2b0a3e3fe54d5614b537bced5fe39c3a9a29